library/ssl_tls.c

```diff
@@ -5655,7 +5655,7 @@ uint16_t mbedtls_ssl_get_tls_id_from_ecp_group_id(mbedtls_ecp_group_id grp_id)
 static const struct {
     uint16_t tls_id;
     const char *name;
-} tls_id_curve_name_table[] =
+} tls_id_curve_group_table[] =
```

Comment:
Ah, sorry, I think my comment wasn't clear: things that only deal with curves don't need to be renamed. It's only for things that deal with both elliptic curves and ffdh parameters that I was recommending using the generic term "group" with encompasses both. So, `curve_name` is fine here, but the table in `ssl_test_lib.c` that holds both curves and ffdh params should be called `group_name` instead.

Reply:
Ops, sorry, that's my fault that I didn't thought what I was doing. Sorry for that. Fixed it ;)